labs/direct-state-transfer_perun.md

```diff
@@ -0,0 +1,62 @@
+# Lab Name
```

Comment:
Are you looking for the lab name (and thus repo) to be named direct-state-transfer, perun, or direct-state-transfer-perun? Based on what you want, please make sure that the .md file matches your expectation.

Reply:
Ok. I am going to discuss this with all involved parties and then change the md file accordingly.

Reply:
I updated the lab name to "Perun Framework" and changed this PR's title and md file accordingly. Cf. 3c8a796